src/extension/wps/src/test/java/au/org/emii/gogoduck/worker/GoGoDuckModuleTest.java

```diff
@@ -1,37 +1,32 @@
 package au.org.emii.gogoduck.worker;
 
-import static org.junit.Assert.assertEquals;
+import au.org.emii.utils.GoGoDuckConfig;
+import org.junit.Before;
+import org.junit.Test;
+import ucar.nc2.Attribute;
+import ucar.nc2.NetcdfFile;
+import ucar.nc2.NetcdfFileWriter;
 
 import java.io.File;
-import java.lang.reflect.Method;
 import java.nio.file.Files;
 import java.nio.file.StandardCopyOption;
 import java.util.ArrayList;
 import java.util.List;
 
-import org.junit.Before;
-import org.junit.Test;
-
-import ucar.nc2.Attribute;
-import ucar.nc2.NetcdfFile;
-import ucar.nc2.NetcdfFileWriter;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 public class GoGoDuckModuleTest {
     public GoGoDuckModule ggdm = null;
+    public GoGoDuckConfig goGoDuckConfig = null;
 
     @Before
     public void beforeEach() {
-        ggdm = new GoGoDuckModule();
-        ggdm.init("", new HttpIndexReader(null, ""), "TIME,1,2;LONGITUDE,2,3", null);
-    }
-
-    public class GoGoDuckModule_test extends GoGoDuckModule {
-        @Override
-        protected List<Attribute> getGlobalAttributesToUpdate(NetcdfFile nc) {
-            List<Attribute> newAttributeList = new ArrayList<Attribute>();
-            newAttributeList.add(new Attribute("test_attribute", "test_value"));
-            return newAttributeList;
-        }
+        goGoDuckConfig = mock(GoGoDuckConfig.class);
+        ggdm = new GoGoDuckModule("", new HttpIndexReader(null, ""), "TIME,1,2;LONGITUDE,2,3", null, goGoDuckConfig);
```

Comment:
HttpIndexreader was required for the command line only I believe - perhaps we can remove it now